### PR TITLE
Add Parsec

### PR DESCRIPTION
**Pull request by jamezrin**
Parsec is a remote access tool that focus primarily on gaming.

### CODE REVIEW

**Review by stephengroat — DISMISSED** [review]
[looking relatively popular on alexa](https://www.alexa.com/siteinfo/parsecgaming.com), approving

**Comment by jamezrin** [comment]
Sorry for asking but is this going to be merged?

File: _data/remote.yml [hunk 1]
```diff
@@ -52,6 +52,14 @@ websites:
       software: Yes
       doc: https://www.syspectr.com/en/faq/how-do-i-activate-deactivate-two-factor-authentication-for-my-account
 
+    - name: Parsec
+      url: https://www.parsecgaming.com
+      img: parsec.png
+      tfa: No
+      twitter: ParsecTeam
+      facebook: parsecgaming
+      email_address: founders@parsec.tv
```
Comment (mxxcon) [inline _data/remote.yml]: Their website lists founders@parsecgaming.com address

**Review by mxxcon — CHANGES_REQUESTED** [review]

**Review by mxxcon — APPROVED** [review]

**Comment by jamezrin** [comment]
Anything else for it to get merged?

**Pull request merged**